pallets/xcmp-handler/src/lib.rs

```diff
@@ -234,8 +237,10 @@ pub mod pallet {
 			let xcm_data = XcmChainCurrencyData::<T>::get(para_id, currency_id)
 				.ok_or(Error::<T>::CurrencyChainComboNotFound)?;
 
-			let (_, target_instructions) = Self::xcm_instruction_skeleton(para_id, xcm_data)?;
+			let (_, target_instructions) =
+				Self::xcm_instruction_skeleton(para_id, xcm_data.clone())?;
```

Comment:
need to understand this line.

Reply:
Because we got rid of the Copy in 
	#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
	pub struct XcmCurrencyData {